src/models/forms/neuburgForm.js

```diff
@@ -17,6 +17,25 @@ export default {
 
   // data on the object
   accommodation: {
+    title: {
+      type: String,
+      validate: {
+        validator: function (value: string): boolean {
+          return value.length >= 10 && value.length <= 40
+        },
+        message: 'Der Titel muss zwischen 10 und 40 Zeichen haben.'
+      },
+      required: [true, 'Titel ist leer.']
+    },
+    location: {
+      type: String,
+      validate: {
+        validator: function (value: string): boolean {
+          return !value || value.length <= 40
```

Comment:
26 is enough here :P

*Schmedeswurtherwesterdeich (26 letters) *

https://en.wikipedia.org/wiki/List_of_long_place_names